.github/workflows/pull-request-jackson.yml

```diff
@@ -30,32 +30,41 @@ jobs:
     name: ITs
     runs-on: ubuntu-latest
     if: contains(github.event.pull_request.labels.*.name, 'pr-jackson')
-    strategy:
-      max-parallel: 4
-      # note the big include entry in matrix is used to ensure we get human readable names for the jobs
-      matrix:
-        include:
-          - jackson-version: 2.6.7
-            target-library: Spark 2.4.4
-            java-version: 11
-          - jackson-version: 2.10.0
-            target-library: Spark 3.0.1
-            java-version: 11
-          - jackson-version: 2.11.4
-            target-library: Spark 3.0.1
-            java-version: 11
-          - jackson-version: 2.12.5
-            target-library: Spark 3.0.1
-            java-version: 11
     steps:
       - uses: actions/checkout@v3
-      - name: Setup Java, Maven, Gradle
+      - name: Setup runner
+        uses: ./.github/actions/setup-runner
+      - name: Setup Java, Gradle
         uses: ./.github/actions/dev-tool-java
+
+      # These four steps are "too quick" to justify a matrix job
```

Comment:
isn't it better to do a matrix job now rather than later, given that newer versions of jackson/spark are unavoidable?

Reply:
yeah but what if the next jackson/spark version comes along? wouldn't we anyway end up doing a matrix job here eventually?

Reply:
I don't get your point.
You have to update the GH WFs, right?